File: template/python3/function/handler.py
import qqbot
from pkg.openapi import OpenAPI

token, sandbox = OpenAPI()
msg_api = qqbot.MessageAPI(token, sandbox)


def handle(data: qqbot.Message):
    send = qqbot.MessageSendRequest("<@!" + data.author.id + ">ok", data.id)
    return msg_api.post_message(data.channel_id, send)

File: template/php/function/Handler.php
<?php

namespace App;

use BotFaas\QQGuild\DTO\Message;
use BotFaas\QQGuild\DTO\MessageToCreate;
use BotFaas\QQGuild\OpenAPI as QQOpenAPI;
use Pkg\OpenAPI;

/**
 * Class Handler
 * @package App
 */
class Handler
{

    public QQOpenAPI $openapi;

    public function __construct()
    {
        $this->openapi = OpenAPI::OpenAPI();
    }

    public function handle(Message $msg)
    {
        return $this->openapi->MessageAPI()->PostMessage($msg->channel_id, new MessageToCreate([
            'content' => "<@!{$msg->author->id}>ok",
            'msg_id' => $msg->id,
        ]));
    }
}
